drivers/memc/memc_stm32.c

```diff
@@ -49,12 +58,21 @@ static int memc_stm32_init(const struct device *dev)
 		return -ENODEV;
 	}
 
-	r = clock_control_on(clk, (clock_control_subsys_t)&config->pclken);
+	r = clock_control_on(clk, (clock_control_subsys_t)&config->pclken[0]);
 	if (r < 0) {
 		LOG_ERR("Could not initialize FMC clock (%d)", r);
 		return r;
 	}
 
+	if (IS_ENABLED(STM32_FMC_DOMAIN_CLOCK_SUPPORT) && (config->pclk_len > 1)) {
```

Comment:
This change is perfectly fine, but you don't provide any target configuration to make use of it. Any reason for not using it?